### PR TITLE
Add `pull_request_target` in base_ref and head_ref doc

### LINKED ISSUE #5457: Missing `pull_request_target` case in base_ref and head_ref ctx
### What is the current behavior?

https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts
> The head_ref or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a pull_request.
> The base_ref or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a pull_request.

### What changes are you suggesting?
The `base_ref` or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run are `pull_request` **and `pull_request_target`**.

 The `head_ref` or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run are `pull_request` **and `pull_request_target`**.

### Additional information

No more info.

### PR DESCRIPTION
**Pull request by Yikun**
### Why:

**Closes: https://github.com/github/docs/issues/5457**

### What's being changed:
The `github.head_ref` and `github.base_ref` are available when the event that triggers a workflow run are `pull_request` and `pull_request_target`.

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by RattaphonKarnsomlap — APPROVED** [review]

**Review by RattaphonKarnsomlap — APPROVED** [review]

File: content/actions/reference/context-and-expression-syntax-for-github-actions.md [hunk 1]
```diff
@@ -93,11 +93,11 @@ The `github` context contains information about the workflow run and the event t
 | `github.action` | `string` | The name of the action currently running. {% data variables.product.prodname_dotcom %} removes special characters or uses the name `run` when the current step runs a script.  If you use the same action more than once in the same job, the name will include a suffix with the sequence number.  For example, the first script you run will have the name `run1`, and the second script will be named `run2`. Similarly, the second invocation of `actions/checkout` will be `actionscheckout2`. |
 | `github.action_path` | `string` | The path where your action is located. You can use this path to easily access files located in the same repository as your action. This attribute is only supported in composite run steps actions. |
 | `github.actor` | `string` | The login of the user that initiated the workflow run. |
-| `github.base_ref` | `string` | The `base_ref` or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a `pull_request`. |
```
Comment (RattaphonKarnsomlap) [inline content/actions/reference/context-and-expression-syntax-for-github-actions.md:96]: "pull_request"

**Review by RattaphonKarnsomlap — COMMENTED** [review]
"pull_request"

File: content/actions/reference/context-and-expression-syntax-for-github-actions.md (hunk 1 above)
Comment (RattaphonKarnsomlap) [inline content/actions/reference/context-and-expression-syntax-for-github-actions.md:96]: | `github.base_ref` | `string` | The `base_ref` or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run are `pull_request` and `pull_request_target`. |

**Review by RattaphonKarnsomlap — CHANGES_REQUESTED** [review]

File: content/actions/reference/context-and-expression-syntax-for-github-actions.md [hunk 2]
```diff
@@ -93,11 +93,11 @@ The `github` context contains information about the workflow run and the event t
...
+| `github.base_ref` | `string` | The `base_ref` or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run are `pull_request` and `pull_request_target`. |
 | `github.event` | `object` | The full event webhook payload. For more information, see "[Events that trigger workflows](/articles/events-that-trigger-workflows/)." You can access individual properties of the event using this context. |
 | `github.event_name` | `string` | The name of the event that triggered the workflow run. |
 | `github.event_path` | `string` | The path to the full event webhook payload on the runner. |
-| `github.head_ref` | `string` | The `head_ref` or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a `pull_request`. |
```
Comment (RattaphonKarnsomlap) [inline content/actions/reference/context-and-expression-syntax-for-github-actions.md:100]: | `github.head_ref` | `สตริง` | "head_ref" หรือสาขาต้นทางของคำขอดึงในเวิร์กโฟลว์ที่เรียกใช้ คุณสมบัตินี้จะพร้อมใช้งานก็ต่อเมื่อเหตุการณ์ที่ทริกเกอร์การรันเวิร์กโฟลว์คือ `pull_request 'และ` pull_request_target ' |

**Review by RattaphonKarnsomlap — CHANGES_REQUESTED** [review]

**Review by RattaphonKarnsomlap — CHANGES_REQUESTED** [review]
P

**Comment by janiceilene** [comment]
@Yikun Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

**Comment by Yikun** [comment]
@janiceilene updated

**Comment by janiceilene** [comment]
@Yikun Thanks so much for opening an issue and filling out the PR template! I'll get this triaged for review ⚡

**Review by dow572 — DISMISSED** [review]

**Comment by nelsondani** [comment]
****_34000_****

**Review by VChernomYA — APPROVED** [review]

File: content/actions/reference/context-and-expression-syntax-for-github-actions.md [hunk 3]
```diff
@@ -93,11 +93,11 @@ The `github` context contains information about the workflow run and the event t
...
 | `github.action_path` | `string` | The path where your action is located. You can use this path to easily access files located in the same repository as your action. This attribute is only supported in composite run steps actions. |
 | `github.actor` | `string` | The login of the user that initiated the workflow run. |
-| `github.base_ref` | `string` | The `base_ref` or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a `pull_request`. |
+| `github.base_ref` | `string` | The `base_ref` or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run are `pull_request` and `pull_request_target`. |
```
Comment (lucascosti) [inline content/actions/reference/context-and-expression-syntax-for-github-actions.md:96]:
```suggestion
| `github.base_ref` | `string` | The `base_ref` or target branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is either `pull_request` or `pull_request_target`. |
```

File: content/actions/reference/context-and-expression-syntax-for-github-actions.md [hunk 4]
```diff
@@ -93,11 +93,11 @@ The `github` context contains information about the workflow run and the event t
...
 | `github.event` | `object` | The full event webhook payload. For more information, see "[Events that trigger workflows](/articles/events-that-trigger-workflows/)." You can access individual properties of the event using this context. |
 | `github.event_name` | `string` | The name of the event that triggered the workflow run. |
 | `github.event_path` | `string` | The path to the full event webhook payload on the runner. |
-| `github.head_ref` | `string` | The `head_ref` or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a `pull_request`. |
+| `github.head_ref` | `string` | The `head_ref` or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run are `pull_request` and `pull_request_target`. |
```
Comment (lucascosti) [inline content/actions/reference/context-and-expression-syntax-for-github-actions.md:100]:
```suggestion
| `github.head_ref` | `string` | The `head_ref` or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is either `pull_request` or `pull_request_target`. |
```

**Review by lucascosti — DISMISSED** [review]
Thanks for this change, @Yikun! I've made a minor wording change, but otherwise this looks great.

**Review by lucascosti — APPROVED** [review]

**Review by dow572 — COMMENTED** [review]
Cogency

**Pull request merged**